apps/admin/backend/test/app.ts

```diff
@@ -131,7 +131,7 @@ export async function configureMachine(
 export function buildMockLogger(
   auth: DippedSmartCardAuthApi,
   workspace: Workspace
-): Logger {
+): MockLogger {
```

Comment:
The result of this function is passed to `testDetectDevices`, which now requires a `MockBaseLogger`.